test/test.ts

```diff
@@ -3,93 +3,67 @@
  */
 import test from 'ava';
 import cbor from 'cbor';
-import {
-  UDPDevice,
-  MockNetwork,
-  RemoteAddressInfo,
-} from '../platform/mock-radio';
+import { UDPDevice, MockNetwork } from '../platform/mock-radio';
 import { loadHomeApp } from '../platform/stub-setup';
 import {
   MockLocalHomePlatform,
   UDPScanConfig,
   ScanState,
 } from '../platform/mock-local-home-platform';
 
+// Common address for all UDP messaging
+const updAddress: string = '255.255.255.255';
+
 // Tests a UDP identify flow end-to-end
 test('udp-device-connects', async (t) => {
   // First, create a scan configuration
-  const scanConfig = new UDPScanConfig(
+  const scanConfig: UDPScanConfig = new UDPScanConfig(
     ScanState.Unprovisioned,
-    '255.255.255.255',
+    updAddress,
```

Comment:
shouldn't we also get rid of this, since we don't send the broadcast anymore?

Reply:
Yeah we can.  Removed.